build/vagrant/install_app.sh

```diff
@@ -9,6 +9,7 @@ ln -fs ../vendor/wmde/fundraising-frontend-content/resources web/resources
 
 npm install
 npm run build-js
+npm run build-assets
```

Comment:
You need to switch the order because `build-assets` creates the `_js` directory that `build-js` uses.